specification/http/src/test/java/org/kaazing/specification/http/rfc7232/IfMatchIT.java

```diff
@@ -0,0 +1,327 @@
+/**
+ * Copyright (c) 2007-2014 Kaazing Corporation. All rights reserved.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/**
+ *  Test to validate behavior of <b>if-match</b> header as specified in
+ *  <a href="https://tools.ietf.org/html/rfc7232">RFC 7232:
+ *   Hypertext Transfer Protocol (HTTP/1.1): Conditional Requests</a>.
+ */
+package org.kaazing.specification.http.rfc7232;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.rules.RuleChain.outerRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.DisableOnDebug;
+import org.junit.rules.TestRule;
+import org.junit.rules.Timeout;
+import org.kaazing.k3po.junit.annotation.Specification;
+import org.kaazing.k3po.junit.rules.K3poRule;
+
+public class IfMatchIT {
+    private final K3poRule k3po = new K3poRule().setScriptRoot("org/kaazing/specification/http/rfc7232/preconditions/if.match");
+    private final TestRule timeout = new DisableOnDebug(new Timeout(5, SECONDS));
+
+    @Rule
+    public final TestRule chain = outerRule(k3po).around(timeout);
+
+    @Test
+    @Specification({
+        "multiple.etags.delete.status.204/request",
+        "multiple.etags.delete.status.204/response" })
+    public void shouldSucceedWithDeleteAndValidETagInTheList() throws Exception {
```

Comment:
Is an ETag only valid if it is matching, and invalid only if it is not matching?
If so, perhaps use the term `MatchingETagInTheList` in the test naming for each test?
Again, we should use the same terminology as the RFC here so that it is clear, and if that's "invalid" then no problem, keep as is.